aquadoggo/src/db/stores/entry.rs

```diff
@@ -15,30 +17,70 @@ use p2panda_rs::storage_provider::traits::{AsStorageEntry, EntryStore};
 use crate::db::models::entry::EntryRow;
 use crate::db::provider::SqlStorage;
 
+#[derive(Debug, Clone, PartialEq)]
+
+pub struct StorageEntry {
+    entry_signed: EntrySigned,
+    operation_encoded: OperationEncoded,
+}
```

Comment:
I removed this previously because technically we don't need two structs (`EntryRow` and `StorageEntry`) to handle entries coming in and out of the db because the storage traits could be implemented directly on `EntryRow`. However, I decided to reintroduce it as I think it improves understanding of what each struct is for and we _do_ need different structs in most other stores, so it matches and will be recognisable across the repo.

Reply:
We need to document this!

Reply:
Related issue here: https://github.com/p2panda/p2panda/issues/320